lib/storage-revocation.js

```diff
@@ -0,0 +1,97 @@
+/*!
```

Comment:
We have `storage-delegations` and `storage-revocation` -- can we normalize? `storage-revocations`?

Reply:
Fixed.